Source/Urho3D/Core/ProcessUtils.cpp

```diff
@@ -540,7 +540,7 @@ String GetLoginName()
 {
 #if defined(__linux__) && !defined(__ANDROID__)
     struct passwd *p = getpwuid(getuid());
-    if (p) 
+    if (p != NULL) 

```

Comment:
I cannot help but notice this inconsistency from our code convention. The former form is the desired convention (https://urho3d.github.io/documentation/HEAD/_coding_conventions.html).

Doing a quick scan using ```git grep``` between current master and 1.6 shows that the IK subsystem is the biggest offender.

```
git grep -w NULL
git grep -w NULL origin/1.6
```